api/types.proto

```diff
@@ -506,6 +506,21 @@ message PortConfig {
 		UDP = 1 [(gogoproto.enumvalue_customname) = "ProtocolUDP"];
 	}
 
+	enum Exposition {
+		option (gogoproto.enum_customname) = "PortConfigExposition";
+		option (gogoproto.goproto_enum_prefix) = false;
+
+		// PortConfigExpositionIngress exposes the port across the cluster on
+		// all nodes.
+		INGRESS = 0;
+
+		// PortConfigExpositionHost exposes the port on just the target host.
+		// If the published port is undefined, an ephemeral port will be
+		// allocated. If the published port is defined, the node will attempt
```

Comment:
Question :
is this task-specific publish-port ?
If it is service-wide publish-port, how can we support multiple tasks on the same host ?